<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectsSelectionDialog</class>
 <widget class="QDialog" name="EffectsSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Damage Chit Effects</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="gunDestroyedBox">
     <property name="title">
      <string>Gun destroyed</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="gun_selection_cmb"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="confirm_btn">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="rudderJamBox">
     <property name="title">
      <string>Rudder Jam</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="jammed_left_radio">
        <property name="text">
         <string>Jammed Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="jammed_right_radio">
        <property name="text">
         <string>Jammed Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="jam_duration">
        <property name="suffix">
         <string> Turns</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="fuelLostBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Fuel Tank Hit</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="fuel_lost">
        <property name="suffix">
         <string> fuel</string>
        </property>
        <property name="prefix">
         <string>lose </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirm_btn</sender>
   <signal>clicked()</signal>
   <receiver>EffectsSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>EffectsSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
